asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
```

Comment:
To relay last night's discussion, keeping request/response here even though the GenericPacket also already implements that is useful to a) have a single way to make it clear if the request sender requires a response or not, and b) avoid the situation where one side sends a response but the other side thinks it's a request, and then responds to it, after which both sides would get confused.